e2e-tests/fixtures/Sequence.ts

```diff
@@ -42,42 +47,60 @@ export class Sequence {
     await this.page.getByRole('button', { name: 'Close' }).click();
 
     this.updatePage(this.page);
-    await this.tableRow.waitFor({ state: 'attached' });
-    await this.tableRow.waitFor({ state: 'visible' });
+    await this.sequenceTable.waitFor({ state: 'attached' });
+    await this.sequenceTable.waitFor({ state: 'visible' });
 
     return this.sequenceName;
   }
 
+  async createWorkspace() {
+    await this.page.locator('button:has-text("Create Workspace")').click();
+
+    this.updatePage(this.page);
+
+    await expect(this.workspaceModal).toBeVisible();
+    await this.workspaceNameTextbox.click();
+    this.workspaceName = uniqueNamesGenerator({ dictionaries: [adjectives, colors, animals] });
+    await this.workspaceNameTextbox.fill(this.workspaceName);
+    await this.page.locator('button:has-text("Save Workspace")').click();
+
+    await expect(this.workspaceModal).not.toBeVisible();
+
+    this.updatePage(this.page);
+    await this.filterTable(this.workspaceTable, this.workspaceName);
+    await this.workspaceTableRow.click();
```

Comment:
I'm not sure that navigating to the workspace here is the right thing for a `createWorkspace` function. Potentially not every test would require that the workspace is immediately navigated to after creation. It might be more appropriate to pass the workspace name into the `createSequence` function and have it navigate to the workspace prior to sequence creation.